spec/features/users/user_edit_profile_spec.rb

```diff
@@ -39,10 +39,10 @@
     expect(page).to have_field(:vegetarian, checked: true)
     expect(page).to have_field(:gluten_free, checked: true)
     expect(page).to have_field(:dairy_free, checked: false)
-    expect(page).to have_field(:keto, checked: false)
+    expect(page).to have_field(:ketogenic, checked: false)
```

Comment:
Good job checking these since their values/titles directly feed into other parts of the app that depend on these being consistent.